db/migrate/20200218155902_add_comments.rb

```diff
@@ -0,0 +1,25 @@
+class AddComments < ActiveRecord::Migration[6.0]
+  def change
+    create_table :comments, if_not_exists: true do |t|
+      t.integer :discussion_id, null: false
+      t.integer :number, null: false
```

Comment:
Consider adding a composite unique index on `discussion_id` and `number`, corresponding to the uniqueness validation, and use the db index to protect the data integrity.